contrib/docker/comdb2-docker-dev

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I'll remove this script.  It's no longer needed - just running the image is enough, and the configuration it spits out can change if stopped/started.